### DISCUSSION TITLE
How to deploy nuxt to my own server?

### DISCUSSION THREAD

**Discussion by xiangnanscu** [nuxt/nuxt#16820] [Questions]
Since the `.output` folder is in `.gitignore` by default, the output of  `nuxt build` in my local machine (wsl2 ubuntu 20.02) can't push to my production server (ubuntu 20.02).

Or should  I run `nuxt build` and `pm2 reload nuxt`  in the git hooks `post-receive` in production server ?

**Comment by CraxTrader** [comment]
nuxt generate for static
